tools/assets/stylesheets/avaiga.css

```diff
@@ -32,6 +47,143 @@ a.ext-link:after {
   font-size: .8em;
 }
 
+.d-flex{
+  display: flex;
+} 
+
+/* ---------- ---------- ---------- TOOLTIP ---------- ---------- ---------- */
+
+.tp-tooltip{
+  position: absolute;
+  bottom: 100%;
+  left: 50%;
+  transform: translate(-50%, -0.5rem);
+  z-index: 1;
+  min-width: 100px;
+  max-width: 200px;
+  padding: 1em;
+  background-color: var(--md-primary-fg-color);
+  border-radius: 0.5em;
+  color: var(--md-primary-bg-color);
+  font-size: 0.9em;
+  text-align: center;
+  opacity: 0;
+  transition: opacity var(--transition-duration-base),
+              transform var(--transition-duration-base),
+              left 0s;
+}
+
+.tp-tooltip img{
+  min-width: 150px;
+  max-width: 100%;
+}
+
+.tp-tooltip::after{
+  content: '';
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translateX(-50%);
+  border: 0.5rem solid transparent;
+  border-top: 0.5rem solid var(--md-primary-fg-color);
+  border-bottom: none;
+}
+
+/* ---------- ---------- ---------- PILLS ---------- ---------- ---------- */
+
+ul.tp-pills-list{
+  margin-left: 0;
+  list-style: none;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+}
+ul.tp-pills-list li {
+  margin: 0;
+  min-width: 0;
+}
+
+.tp-pill,
+a.tp-pill{
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  text-align: center;
+  padding: 0.6em 1.5em;
+  border-radius: 50rem;
+  background: var(--md-paper-bg-color);
+  color: var(--md-default-fg-color);
+}
+
+.tp-pill .tp-tooltip{
+  left: -9999rem;
+  transform: translate(-50%, 0);
+}
+
+.tp-pill:hover .tp-tooltip{
+  left: 50%;
+  transform: translate(-50%, -0.5rem);
+  opacity: 1;
+  transition: opacity var(--transition-duration-base) var(--tooltip-show-delay),
+              transform var(--transition-duration-base) var(--tooltip-show-delay),
+              left 0s var(--tooltip-show-delay);
+}
+
+/* ---------- ---------- ---------- Content Cards ---------- ---------- ---------- */
+
+.tp-content-card{
+  flex: 0 0 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: stretch;
+  padding: 1.25em;
+  border-radius: 0.5em;
+  background: var(--md-paper-bg-color);
```

Comment:
May be a slight skew in the background color would make them look better.

Reply:
Done